IPython/external/qt_loaders.py

```diff
@@ -110,17 +110,26 @@ def has_binding(api):
                    QT_API_PYQT_DEFAULT: 'PyQt4'}
     module_name = module_name[api]
 
-    import imp
+
+    if sys.version_info >= (3, 4):
+        def find_module(name, path):
```

Comment:
Since we only need to check if the module exists, I think we can do something simpler based on [find_loader](https://docs.python.org/3/library/importlib.html#importlib.find_loader).

The way we're using it, it also needs to raise if the module is not found, whereas I think this just returns None if there's no match.